docs/adr/ADR-01-STRUCTURED_DATA.md

```diff
@@ -0,0 +1,148 @@
+* PropertyName: Name of the property of an entry. Can only consist of letters, numbers and "." TODO is that ok? do we want to allow other special chars?
+* VariantInformation: Specifying what kind of variant this key is. Format is "VariantName"="VariantValue"
+
+Currently supported variants are:
+* Language: which language a value is in
+* Format: what kind of format the value is in
+
+If a certain variant makes sense for a certain property needs to be decided per property basis.
+
+#### Language Variant
+
+Has the VariantName "lang" and the value is a two-letter language abbreviation, like "en", "de", "fr", ...
+
+TODO define what that means for searching, for displaying, how languages are chosen and so on
+
+#### Format Variant
+
+Has the VariantName "format" and for the value see the [Formats](#formats) section.
+
+TODO define more stuff here?
+
+
+## Formats
+
+We decided to support following formats for now:
+* Numbers (VariantValue "number")
+* Dates (VariantValue "date")
+* Lists (VariantValue "list")
+* Markdown (VariantValue "markdown")
+
+### Numbers
+
+Knowledge about numbers is mostly important to correctly sort them.
+
+The format looks like: TODO define format and possible numbers here
+
+Examples:
+* 0
+* 1
+* -5
+* ...?
+
+### Dates
+
+Knowledge about dates is important for before/after + duration searches and to sort them correctly.
+
+The format is the "Date and time with the offset" format of the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard.
+
+Examples:
+* 2024-04-27T23:59:00+02:00
+* 2024-04-27T11:00:00Z
+
+### Lists
+
+Knowledge about lists is mostly important for contains searches and search faceting. We cannot sort lists. (TODO can/should we?)
+
+The format is a comma "," seperated list of values. That means that commas occurring in a value have to be escaped by a backslash "\" and occurring backslashes also have to be escaped by a backslash.
+
+Examples:
+* value1 (this is a list consisting only of one value "value1")
+* value1, value2 (this is a list consisting of two values "value1" and "value2")
+* val\\\\ue1,val\\\,ue2 (this is a list consisting of two values with escaped characters in them "val\\ue1" and "val,ue2") (this example is wrong when looked at directly, please only look at it rendered through an markdown engine, because there are too many "\\" in the unrendered text)
+* (an empty list)
+* value1,,value2 (this is a list consisting of three values "value1", "" (an empty value) and "value2")
+* value1, value2 (this is a list consisting of two values "value1" and " value2" (please note the preceding whitespace in the second value))
+* , (this is a list consisting of two empty values, please note that there is no way to specify a list with one empty value)
+
+### Markdown
+
+Knowledge about markdown is important for rendering it correctly.
+
+The format is normal [Markdown](https://en.wikipedia.org/wiki/Markdown).
```

Comment:
I would use a different wording. 

e.g. "We agree to use standard markdown according to wiki"